lib/core/networking/websockets/middleware.dart

```diff
@@ -0,0 +1,83 @@
+import '../protocol/authentication.dart';
+import '../protocol/protocol.dart';
+
+import 'session.dart';
+
+//* Type declarations
```

Comment:
This would impair visual hierarchy. The empty line would make it seem like something between imports and code, when it is actually the heading for the following code.

Reply:
It looks like it would belong to the following doc comment. Especially when you turn of the highlighting of `//*`

Reply:
Outdated